istio-telemetry/grafana/templates/configmap.yaml

```diff
@@ -27,9 +26,9 @@ data:
         timeInterval: 5s
       editable: true
 
-{{- if .Values.grafana.datasources }}
+{{- if $datasources.datasources }}
     -
-  {{- range $key, $value := .Values.grafana.datasources }}
```

Comment:
@douglas-reid what do you think of how this works? The problem is we need to be able to set the prometheusNamespace now, which is why we have the hardcoded source above.